src/render/draw_background.js

```diff
@@ -42,7 +42,7 @@ function drawBackground(painter: Painter, sourceCache: SourceCache, layer: Backg
         painter.tileExtentPatternVAO.bind(context, program, painter.tileExtentBuffer, []);
     } else {
         program = painter.useProgram('background');
-        gl.uniform4fv(program.uniforms.u_color, [color.r, color.g, color.b, color.a]);
+        gl.uniform4fv(program.uniforms.u_color, ([color.r, color.g, color.b, color.a]: Array<number>));
```

Comment:
These are needed because Flow doesn't like the ambiguity caused by https://github.com/facebook/flow/pull/5893 (👋 @lbud !)

Reply:
This particular case could be written `gl.uniform4f(program.uniforms.u_color, color.r, color.g, color.b, color.a)` (`4f`, without the `v`).